dependencies.yaml

```
@@ -475,12 +475,9 @@ dependencies:
           - numba>=0.57
           - *numpy
           - rapids-dask-dependency==24.8.*,>=0.0.0a0
-          - ucx-py==0.39.*
       - output_types: conda
         packages:
-          - &ucx_py_conda ucx-py==0.39.*
-      - output_types: pyproject
-        packages:
+          - &ucx_py_conda ucx-py==0.39.*,>=0.0.0a0
           - &pylibraft_conda pylibraft==24.8.*,>=0.0.0a0
```

Swap the order of these so it matches the lines below this.
```suggestion
          - &pylibraft_conda pylibraft==24.8.*,>=0.0.0a0
          - &ucx_py_conda ucx-py==0.39.*,>=0.0.0a0
```

---

ah yeah I agree, thanks